.github/workflows/linux.yml

```diff
@@ -184,13 +191,13 @@ jobs:
     runs-on: ubuntu-20.04
     steps:
       - name: Checkout code
-        uses: actions/checkout@v2
+        uses: actions/checkout@v4
         with:
           repository: 'adacore/alt-ergo'
           ref: 'fsf-14'
 
       - name: Use Setup OCaml {{ matrix.ocaml-version }}
-        uses: ocaml/setup-ocaml@v2
+        uses: ocaml/setup-ocaml@v4
```

Comment:
There is no `ocaml/setup-ocaml@v4` yet. `ocaml/setup-ocaml@v3` is the latest for now.